python/ray/autoscaler/aws/example-cloudwatch.yaml

```diff
@@ -0,0 +1,85 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: cloudwatch
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+    # Start by referencing Unified CloudWatch Agent json config file path here.
+    # Note that this config assumes Unified CloudWatch Agent is installed on every cluster node.
+    # For AMI with Unified CloudWatch Agent pre-installed, see https://github.com/amzn/amazon-ray.
+    cloudwatch:
+        # For updates to existing Unified CloudWatch Agent json config file:
+        # Config file is uploaded to the AWS Systems Manager (SSM) Parameter Store during `ray up`.
+
+        # Note only parameters with "AmazonCloudWatch-" prefix are allowed for access with CloudWatchAgentAdminPolicy.
+        # See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-iam-roles-for-cloudwatch-agent.html for additional details.
```

Comment:
I think we can delete these lines since it's describing an internal implementation detail most users won't need to worry about, and I'd think that anyone who manually messes around with these parameter names should expect things to break for one reason or another anyways.